src/main/java/org/folio/rest/impl/util/OkapiResponseUtil.java

```diff
@@ -0,0 +1,47 @@
+package org.folio.rest.impl.util;
+
+import io.vertx.core.AsyncResult;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.jaxrs.model.Error;
+import org.folio.rest.jaxrs.model.Errors;
+
+/**
+ * Utility methods for Okapi Responses.
+ */
+public class OkapiResponseUtil {
+
+  private OkapiResponseUtil() {
+    throw new UnsupportedOperationException("Do not instantiate");
+  }
+
+  public static String getErrorMessage(AsyncResult<Response> reply) {
+    String message = null;
+
+    if (reply.succeeded() && reply.result().getStatus() >= 400
+      && reply.result().getStatus() < 500 && reply.result().hasEntity()) {
```

Comment:
Should 500 also be considered, i.e. `<=500`?

Reply:
I changed it from < 600 because I thought we weren't handling 5xx issues.

The reason why it is no longer handling the 500 issue is due to the change in how it is handling the UUID test. The 500 appears to have only happened on invalid UUID, which was not what the test was for. When we write a new test, then the 500 will need to be handled.

Should this utility handle all of the 5xx codes, the 4xx an 500, or just the 4xx codes?

Reply:
Hmm... that's a good question. As far as I understand the method is mostly used to check the 'edge' cases, like same position in queue, constraint violation, etc., right? 

If so, then I think it is fine to use only 4xx, since you're expecting that constraint violation is a 4xx status code, and a 5xx code is most likely is not the case we're trying to handle.